sw/ground_segment/cockpit/sectors.ml

```diff
@@ -32,7 +32,7 @@ let load = fun geomap () ->
       None -> ()
     | Some f ->
       try
-        let xml = Xml.parse_file f in
+        let xml = ExtXml.parse_file f in
```

Comment:
not sure this one should be changed, it breaks the `try ... catch` and the popup message